app/input_objects/pages/update_exit_page_input.rb

```diff
@@ -2,6 +2,7 @@ class Pages::UpdateExitPageInput < BaseInput
   attr_accessor :form, :page, :record, :exit_page_markdown, :exit_page_heading
 
   validates :exit_page_heading, :exit_page_markdown, presence: true
+  validates :exit_page_heading, :exit_page_markdown, length: { maximum: 5000 }
```

Comment:
We are adding a new validation to an existing field, do we need to think about the forms people have already created?

Exit pages is behind a flag so I don't think it will matter? The worst thing which will happen is that someone won't be able to edit their exit page if they have already made a really long one.

I guess we also don't need to worry that someone has put "unsupported" markdown features in the current text box.